docs/user-guide/cli-installcli.md

```diff
@@ -29,19 +29,24 @@ If you do not have internet access at your site, use the following method to ins
 
         **Tip:** npm is included with the Node.js installation. Issue the command `npm --version` to verify the version of npm that is installed.
 
-    - **(Optional)** If you plan to install plug-ins, review the [Software requirements for CLI plug-ins](cli-swreqplugins.md).
+2. Address the software requirements for Secure Credential Storage and CLI plug-ins.
+
+   - On graphical Linux, install `gnome-keyring` and `libsecret` on your computer.
+
+   - There are additional requirements for headless Linux systems. See the [SCS plug-in Readme](https://docs.zowe.org/stable/user-guide/cli-swreqplugins.html) for details.
```

Comment:
Should the link title be "Software requirements for CLI plug-ins"?

Reply:
@tjohnsonBCM I think we can link straight to the readme. But since seeing this comment I actually reorganized things a bit: 
- Step one is only about the base CLI 
- Step 2 is for Linux users only 
- Plug-in reqs are now an explicit optional step 7 immediately before the install plugins step, so that it isn't missed. 

I also reflected the changes to the online registry section. Sry about the late changes.